pytensor/link/numba/dispatch/basic.py

```diff
@@ -74,7 +74,7 @@ def numba_njit(*args, fastmath=None, **kwargs):
         message=(
             "(\x1b\\[1m)*"  # ansi escape code for bold text
             "Cannot cache compiled function "
-            '"(numba_funcified_fgraph|store_core_outputs)" '
+            '"(numba_funcified_fgraph|store_core_outputs|nb_cholesky)" '
```

Comment:
why cholesky specifically?

Reply:
It was emitting this warning in a test, strangely the other lapack functions were not? Gotta double check that

Reply:
I assume none of them cache. I thought the tests all had a pytest.filterwarnings decorator on them

Reply:
So it was the case that the warning was only making it fail with the cholesky, because we had a `pytest.mark.filterwarnings("error")` in the blockwise test and we tested a blockwise with cholesky. Instead of ignorning I am also making it fail if warnings are emitted on the slinalg test, and thus I am filtering those other Ops here as well.